pkgs/applications/misc/sequeler/default.nix

```diff
@@ -22,7 +22,7 @@ in stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ meson ninja pkgconfig pantheon.vala gettext wrapGAppsHook python3 desktop-file-utils ];
 
-  buildInputs = [ gtk3 glib pantheon.granite libgee sqlGda gtksourceview libxml2 libsecret libssh2 ];
+  buildInputs = [ gtk3 gnome3.adwaita-icon-theme glib pantheon.granite libgee sqlGda gtksourceview libxml2 libsecret libssh2 ];
```

Comment:
This reflects what I think about this
* https://github.com/NixOS/nixpkgs/pull/43150#issuecomment-450060363

Users should probably just install the proper icon theme.

Reply:
Hmm, I've not seen that thread. I've just done that on my local system.

I just noted that a change like this was done to the gparted package and that that package had working icons nowadays.

Reply:
I actually made that change since when I updated the installer images to use a non-root user.
Doesn't seen plasma5 was configured to include an icon set, or is and something's up so I didn't want to regress the appearance of gparted in the installer.

Reply:
Makes sense, I think it regressed already when it became gtk3 though, just good that it was caught before 19.09 :)